const errorMiddleware = (err, req, res, next) => {
    console.error(err.stack);
    res.status(500).json({ error: 'Ocurri√≥ un error interno en el servidor' });
  };
  
  module.exports = errorMiddleware;